src/App.ts

```diff
@@ -494,14 +499,14 @@ export default class App {
   ): void;
   public shortcut<
     Shortcut extends SlackShortcut = SlackShortcut,
-    Constraints extends ShortcutConstraints<Shortcut> = ShortcutConstraints<Shortcut>,
```

Comment:
these are prettier changes I believe

Reply:
perhaps, these are the changed by your editor (VS Code). You can manually run `npx prettier -w src`